lib/msteams_hermes/components/adaptive_card.rb

```diff
@@ -12,9 +12,9 @@ module Components
     # https://adaptivecards.io/explorer/AdaptiveCard.html
     ##
     class AdaptiveCard < Base
-      attr_reader :body, :actions, :entities
+      attr_reader :body, :actions, :entities, :width
 
-      def initialize(body:, actions: nil, entities: [])
+      def initialize(body:, actions: nil, entities: [], full_width: false)
```

Comment:
I'm not a fan of the parameter name. I see that `full_width` is mapped/converted [here](https://github.com/xing/msteams_hermes/pull/6/files#diff-dd502871410c713e9ab4dd5edfed3a05eae5d93a573f6177b6322773cfead4eaR46), so the main value is a string called `full` or `default`.
What I would like to see is differentiating this new API from the API/params that are used by the MSTeams endpoint. The parameter changed just slightly by adding the prefix `full`. 
However, the word full is used by Microsoft, [here](https://github.com/xing/msteams_hermes/pull/6/files#diff-dd502871410c713e9ab4dd5edfed3a05eae5d93a573f6177b6322773cfead4eaR47), so you combined the key + (potential) value, but converted it into a bool.

So either you use something like:
`use_full_width`, `use_max_width`, `stretch_card_to_max_width`
or
you let the used assign a symbol, which you define as a constant/enum.

```suggestion
      def initialize(body:, actions: nil, entities: [], width: :default)
```
where width can be (as of now), `default` and `full`.
You may need a module:
```
module MSTAdaptiveCardWidth
  DEFAULT = 'default'
  FULL = 'full'

  def self.all
    [DEFAULT, FULL]
  end
end
```

Then you can do `MSTAdaptiveCardWidth::DEFAULT`, while potentially even omit the `MSTAdaptiveCardWidth` prefix, if you define the module in this very same file.

The advantage is of course that you can basically delete the method `set_width(full_width:)`.
On top of that extending this class to support a new size would be trivial.